gemfire-core/src/main/java/com/gemstone/gemfire/internal/LinuxProcFsStatistics.java

```diff
@@ -236,14 +236,14 @@ private static void getLoadAvg(double[] doubles, int[] ints) {
       if (threadLoad != null) {
         int slashIndex = threadLoad.indexOf('/');
         if (slashIndex >= 0) {
-          try {
-            ints[LinuxSystemStats.threadsINT] = Integer.parseInt(
-                threadLoad.substring(slashIndex + 1));
-          } catch (NumberFormatException ignored) {
-          }
+          threadLoad = threadLoad.substring(slashIndex + 1);
+        }
+        try {
+          longs[LinuxSystemStats.totalThreadsLONG] = Long.parseLong(threadLoad);
+        } catch (NumberFormatException ignored) {
```

Comment:
Yes, the first field is the number of current threads/processes being scheduled. However, it is an instantaneous value and will be misleading by itself. That is why the first three fields in of last 1, 5, 15 minutes average are useful which is exactly the average of this field over those intervals.

About last field, it is just the last allocated PID. It will keep increasing then reset back once max pid limit is reached. The total processes/threads is captured by the second value after '/' so new ones coming up can be easily seen by using "Per Second" option in VSD.